server/tests/test_manual_data_input.py

```diff
@@ -216,7 +216,7 @@ def test_agegroups_existing_data(vanilla_user_client):
         response = vanilla_user_client.post(url, data=data)
         assert response.status_code == 403
     ind.refresh_from_db()
-    assert ind.data.filter(period=period).count() == 4
+    assert ind.data.filter(period=period, is_deleted=False).count() == 4
```

Comment:
This is a good place for an explicit check a refreshed `record_51_75` has `is_deleted` and `deleted_time`